content/resources/openhwtv/s02e01.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "OpenHW TV S02 E01"
+date: 2021-01-28
+season: 2
+episode_title: embedded.com interview and Season One round-up
+---
+
+{{< youtube "LZ8N3uLD4wY" >}}
+
+2021.New year, new episode of OpenHW TV. In the first episode of Season Two, we feature an interview between embedded.com editor-in-chief Nitin Dahad and OpenHW Group President and CEO Rick O'Connor. They discuss the latest milestones of the OpenHW Group, the roadmap for the organization, as well as the key issues that are driving the growth of open-source processor design and development. Plus there's a brief bonus round-up of season one showing the interesting range of guidance and discussion that was presented last year.
```

Comment:
Small nitpick, add a space between `2021.` and `New` at the start of this line

Reply:
thanks for pointing, actually got one more updates including this one from NaylDSouza